### PR TITLE
New maps design

### PR DESCRIPTION
**Pull request by patrickfeltes**
Folds existing indoor and outdoor maps into one fragment with new design.

The bottom bar was covering up the distance and time to walk feature, so I just removed it. I'll talk to design to see if they want to keep it.

Waiting for images for the indoor maps, so right now it's just the old design.

Screenshot:
maps

### CODE REVIEW

File: app/src/main/java/org/hackillinois/android/view/maps/MapsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+        outdoorMapsButton = view.findViewById(R.id.outdoorButton)
+        indoorMapsButton = view.findViewById(R.id.indoorButton)
+        contentFrame = view.findViewById(R.id.mapsContainer)
+
+        outdoorMapsButton.setOnClickListener(outdoorMapsClickListener)
+        indoorMapsButton.setOnClickListener(indoorMapsClickListener)
+        view.tabLayout.addOnTabSelectedListener(tabListener)
+
+        outdoorMapsButton.isSelected = true
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/maps/MapsFragment.kt]: I think we should switch these two. Indoor maps should be to the left of outdoor maps, and indoor maps should start as selected.

Comment (bstrauch24) [reply app/src/main/java/org/hackillinois/android/view/maps/MapsFragment.kt]: Reasoning is that indoor maps load faster than outdoor maps

File: app/src/main/java/org/hackillinois/android/view/maps/OutdoorMapsFragment.kt [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    }
+
+    fun onTabReselected(index: Int) {
+        map?.animateCamera(CameraUpdateFactory.newLatLngZoom(BUILDING_LOCATIONS[index], ZOOM.toFloat()))
+    }
+
+    companion object {
+        private val ZOOM = 18
+
+        private val BUILDING_NAMES = arrayOf(
```
Comment (bstrauch24) [inline app/src/main/java/org/hackillinois/android/view/maps/OutdoorMapsFragment.kt]: We probably don't need these anymore

File: app/src/main/res/layout/fragment_maps.xml [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/outdoorButton"
+        app:tabIndicatorColor="@color/colorAccent"
+        app:tabIndicatorHeight="3dp"
+        app:tabSelectedTextColor="@color/colorAccent"
+        app:tabTextAppearance="@style/TabTextAppearance"
+        app:tabTextColor="@color/colorPrimary"
+        >
+
+        <com.google.android.material.tabs.TabItem
```
Comment (bstrauch24) [inline app/src/main/res/layout/fragment_maps.xml]: Let's put these in alphabetical order? Easier to coordinate with iOS that way

File: app/src/main/res/layout/fragment_indoor_maps.xml [hunk 4]
```diff
@@ -1,58 +1,22 @@
...
+    <ScrollView
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:orientation="vertical"
-        android:background="@color/colorPrimaryLight">
+        android:layout_margin="20dp">
 
-        <com.google.android.material.tabs.TabLayout
-            android:id="@+id/indoor_tabs"
+        <com.davemorrissey.labs.subscaleview.SubsamplingScaleImageView
```
Comment (bstrauch24) [inline app/src/main/res/layout/fragment_indoor_maps.xml]: Does this handle zooming, or device resolution?

Comment (patrickfeltes) [reply app/src/main/res/layout/fragment_indoor_maps.xml]: I think zooming and extremely large images

**Pull request merged**